fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1471,6 +1602,21 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 				if (SERVER_KNOBS->ENABLE_DD_PHYSICAL_SHARD) {
 					ASSERT(rd.dataMoveId.isValid());
 				}
+				auto f = self->dataMoves.intersectingRanges(rd.keys);
```

Comment:
In the future we could ASSERT that the intersecting DataMove in self->dataMoves are all invalid, after we fix the split brain issue? Can you add a comment here?